kvs/client/python/anna/ipc_client.py

```diff
@@ -81,13 +80,16 @@ def get(self, refs):
             for tp in resp.tuples:
                 if tp.error == 1:
                     logging.info('Key %s does not exist!' % (key))
-                    return None
+                    kv_pairs[tp.key] = None
 
                 if tp.lattice_type == LWW:
                     val = LWWValue()
                     val.ParseFromString(tp.payload)
 
-                    kv_pairs[tp.key] = LWWPairLattice(val.timestamp, val.value)
+                    if b'ERROR' in val.value:
```

Comment:
when can this happen?

Reply:
This was an old debug check. Will remove.